### PR TITLE
python3Packages.terminaltexteffects,terminaltexteffects: init at 0.10.1

### PR DESCRIPTION
**Pull request by eihqnh**
## Description of changes
_TerminalTextEffects (TTE) is a terminal visual effects engine._
https://github.com/ChrisBuilds/terminaltexteffects

 init `python3Packages.terminaltexteffects` , `terminaltexteffects`  at 0.10.1

 add qwqawawow (me) as the maintainer

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/terminaltexteffects/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "terminaltexteffects";
+  version = "0.10.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
```
Comment (pbsds) [inline pkgs/development/python-modules/terminaltexteffects/default.nix:16]:
```suggestion

   # no tests on pypi, no tags on github
```

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -16971,6 +16971,12 @@
     githubId = 2141853;
     name = "Bang Lee";
   };
+  qwqawawow = {
+    email = "eihqnh@outlook.com";
+    github = "qwqawawow";
+    githubId = 40905037;
+    name = "qwqawawow";
+  };
```
Comment (pbsds) [inline maintainers/maintainer-list.nix:16979]: please split of this change into a separate commit titled `maintainers: add qwqawawow`

**Review by pbsds — COMMENTED** [review]
Some git-history rework is needed, please squash the update commit, and separate out the maintainer addition

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (qwqawawow) [reply maintainers/maintainer-list.nix:16979]: Corrected.Thank you very much. I didn't notice the guide....

Comment (qwqawawow) [reply maintainers/maintainer-list.nix:16979]: Corrected.Thank you very much. I didn't notice the guide....
